routes/api/vote.ts

```diff
@@ -34,10 +39,22 @@ async function sharedHandler(
       status = 204;
       await deleteVote(vote);
       break;
-    case "POST":
+    case "POST": {
       status = 201;
       await createVote(vote);
+
+      if (item!.userId != user!.id) {
```

Comment:
```suggestion
      if (item!.userId !== user!.id) {
```
Hm, I would've thought that checking for `item === null || user === null` above would've negated the need for the need for the non-null assertion operator `!`.